src/kinvey.d.ts

```diff
@@ -26,13 +26,14 @@ export namespace Kinvey {
 
   // ClientConfig interface
   interface ClientConfig {
-    apiHostname?: string,
-    micHostname?: string,
-    appKey: string,
-    appSecret?: string,
-    masterSecret?: string,
-    encryptionKey?: string,
-    defaultTimeout?: number
+    apiHostname?: string;
+    micHostname?: string;
+    liveServiceHostname?: string;
```

Comment:
I think this shouldn't be required anymore.

Reply:
Or maybe it is - I'm not sure. But if we expect customers with single tenant instances to set the live service URL, we need to document it on devcenter.

Reply:
Probably best to document in the descanter.

Reply:
Add a ticket / PR?

Reply:
I actually deleted this from the typings on my PR (about definitions for LiveService). I thought this was the outdated live service? Now LiveService uses the hostname for other services.